### DISCUSSION TITLE
not able to start the frappe site using fm start

### DISCUSSION THREAD

**Discussion by advent0shafi** [rtCamp/Frappe-Manager#156] [Q&A] [answered]
i try to start the fm start for an site in terminal but it say Exception : [Errno 13] Permission denied: '/home/advent0shafi/frappe/sites/demo.localhost/workspace/frappe-bench/sites/common_site_config.json'
why does this happens ? my version  0.12.0

**Comment by Xieyt** [comment, marked as answer]
@advent0shafi Please use the latest version and LMK if it doesn't works for you.
